### ISSUE TITLE
[feature request] Need QuantizedFusedBatchNorm 

### ISSUE THREAD

**Issue by passerbyd** [tensorflow/tensorflow#12523]
There is no QuantizedFusedBatchNorm operator. The graph transform tools treats FusedBatchNorm operator with fold_old_batch_morm. But not all bns are ready to be folded. This will leave many isolated un-quantized fbn ops in the graph.

**Comment by andydavis1** [comment]
@petewarden Can you comment on this?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 257 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 273 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 288 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 303 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 318 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 333 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 348 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by cy89** [comment]
@suharshs could you please take a look, and comment or redirect?

**Comment by suharshs** [comment]
Hi, quantization efforts have been prioritized for TensorFlow Lite.

Besides that quantized unfused batch norm can result in accuracy issues that we haven't explored fully yet, especially if the entire model is being run in uint8. That being said we will be releasing more tools to quantize TensorFlow Lite models that should address and avoid this eventually.
